distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -27,14 +27,33 @@ Note that the `SAML` and `OIDC` authentication types require session storage to
 
 === Authentication Types
 
-As you add REST endpoints, you may need to add different types of authentication through the Web Context Policy Manager.
+Through the Web Context Policy Manager, authentication types for ${branding} may be configured with the following format:
 
-Any web context that allows or requires specific authentication types should be added here with the following format:
+----
+<AUTH_TYPE>|<AUTH_TYPE|...
+----
+
+Authentication types can be configured separately for:
+* web pages (these are all context paths that do not start with `/services` such as `/admin` and `/search`)
+* endpoints (these are all context paths that start with `/services`)
+
+Configuring separate authentication types for web pages or endpoints is supported through the `web-context-policy-config.properties` configuration file.
+* Navigate to `${home_directory}/etc/`
+* Edit the `web-context-policy-config.properties` file
+* Restart ${branding}
+
+The `web-context-policy-config.properties` file format is:
 
 ----
-/<CONTEXT>=<AUTH_TYPE>|<AUTH_TYPE|...
+/<CONTEXT>=<AUTH_TYPE> then <AUTH_TYPE then ...
 ----
 
+[WARNING]
+====
+If the file-based configuration is used, the *authentication type* configuration for web pages and endpoints in the ${admin-console} is ignored.
```

Comment:
It would be confusing to pull configurations from two different sources. It makes more sense to pick and use one. Hopefully, no one uses the file-based one and we can get rid of it.